.github/workflows/testing-build.yml

```diff
@@ -0,0 +1,30 @@
+name: Testing build (on PR)
+
+on:
+    pull_request:
+        branches: [ main ]
+        types: [ labeled, opened, synchronize, reopened ]
+
+jobs:
+    testing_build:
+        runs-on: ubuntu-latest
+        if: contains(github.event.pull_request.labels.*.name, 'testers needed')
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-java@v4
+              with:
+                  java-version: 17
+                  distribution: 'temurin'
+            - name: Build debug APK
+              run: ./gradlew assembleCoreDebug
```

Comment:
Let's build the `prepaid` flavor as color customization is locked in `core` and also because someday we will remove `core` altogether.